pkg/virtctl/usbredir/usbredir.go

```diff
@@ -175,16 +175,14 @@ func (usbredirCmd *usbredirCommand) Run(command *cobra.Command, args []string) e
 		usbredirExecResChan <- err
 	}()
 
-	sigStopChan := make(chan struct{}, 1)
+	interrupt := make(chan os.Signal, 1)
 	go func() {
-		defer close(sigStopChan)
-		interrupt := make(chan os.Signal, 1)
```

Comment:
Should the `<-interrupt` and `go` be dropped as well?

Reply:
Indeed! For some reason I thought signal.Notify() would block. Thanks!